aspnetcore/fundamentals/websockets/samples/1.x/WebSocketsSample/Startup.cs

```diff
@@ -81,6 +81,7 @@ private async Task Echo(HttpContext context, WebSocket webSocket)
             {
                 await webSocket.SendAsync(new ArraySegment<byte>(buffer, 0, result.Count), result.MessageType, result.EndOfMessage, CancellationToken.None);
 
+                buffer = new byte[1024 * 4];
```

Comment:
Use `Array.Clear` so you don't need to allocate a whole new buffer every loop

Reply:
That is, indeed, much better.
Thanks for pointing this out.

    Array.Clear(buffer, 0, result.Count);